backend/src/utils/constants.ts

```diff
@@ -38,5 +38,9 @@ export const blankDashboardCR: DashboardConfig = {
       disableAppLauncher: false,
       disableUserManagement: false,
     },
+    groupsConfig: {
+      adminGroups: 'dedicated-admins',
```

Comment:
Small nit, `dedicated-admins` is a group created in OpenShift dedicated by default but not in other environments. Can we use something more generic like `dashboard-admins` or `odh-admins`?